### PR TITLE
Feature/add new idea

### PR DESCRIPTION
**Pull request by alxx88**

### CODE REVIEW

File: src/main/webapp/css/ideaDetails.css [hunk 1]
```diff
@@ -26,7 +26,7 @@
 	width: 100% auto;
 	height: 35%;
 	margin: 0 auto;
-	border: black solid thin;
+	/*border: black solid thin;     /*���������� ����������*/
```
Comment (Hanna-Aliakseichykava) [inline src/main/webapp/css/ideaDetails.css]: Please, rewrite comments in English

Comment (alxx88) [inline src/main/webapp/css/ideaDetails.css]: Yes. Sorry forgot remove.

File: src/main/webapp/pages/details.html [hunk 2]
```diff
@@ -1,36 +1,68 @@
-<div id="mainDiv" ng-controller="detailsCtrl as detailsCtrl">
+<div id="mainDiv">
```
Comment (Hanna-Aliakseichykava) [inline src/main/webapp/pages/details.html]: It seems like there are not much general code for showing details and creating new idea. May be it makes sense to split this view into 2?

File: src/main/webapp/js/app.js [hunk 3]
```diff
@@ -56,10 +56,45 @@ angular
...
+				}
+			},
+            onEnter:  function(){ ymaps.ready(mapInit)},
+            parent: 'root'
+        }).
+        state('ideaAddNew', {
+            url: '/ideaAddNew:idea',
+            views: {
+              'main@': {
+                  templateUrl: 'pages/details.html',
```
Comment (Hanna-Aliakseichykava) [inline src/main/webapp/js/app.js]: It seems a little bit illogical to use details view for creating idea

**Comment by Hanna-Aliakseichykava** [comment]
:+1:

**Comment by reshetnev** [comment]
:+1:

**Comment by LostFan** [comment]
:+1:

**Pull request merged**